src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -708,7 +771,14 @@ void RadioSpinel<InterfaceType>::HandleValueIs(spinel_prop_key_t aKey, const uin
             }
 
             otLogInfoPlat("RCP reset: %s", spinel_status_to_cstr(status));
-            mIsReady = true;
+            sIsReady = true;
+        }
+        else if (status == SPINEL_STATUS_SWITCHOVER_DONE || status == SPINEL_STATUS_SWITCHOVER_FAILED)
+        {
+            if (mCallbacks.mSwitchoverDone != nullptr)
+            {
+                mCallbacks.mSwitchoverDone(mInstance, status == SPINEL_STATUS_SWITCHOVER_DONE);
```

Comment:
Where do we set callback from?

Reply:
callback is set with void RadioSpinel<InterfaceType>::SetCallbacks(const struct RadioSpinelCallbacks &aCallbacks), but currently it is not used by ot-daemon.